src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -27,10 +28,21 @@ export async function getIridaWorkflowDescription(workflowUUID) {
  * Get a listing of all Pipelines in IRIDA.
  * @returns {Promise<AxiosResponse<any> | never>}
  */
-export const fetchIridaAnalysisWorkflows = async function() {
+export const fetchIridaAnalysisWorkflows = async function () {
   var ajaxUrl = URL;
   if (window.PAGE.automatedProject !== null) {
     ajaxUrl = `${ajaxUrl}?automatedProject=${window.PAGE.automatedProject}`;
   }
-  return axios.get(ajaxUrl).then(response => response.data);
+  return axios.get(ajaxUrl).then((response) => response.data);
 };
+
+/**
+ * Get details about a specific pipeline to be able to launch.
+ * @param id - UUID identifier for the pipeline
+ * @returns {*}
+ */
+export const getPipelineDetails = ({ id }) =>
+  axios
+    .get(`${AJAX_URL}/${id}`)
+    .then(({ data }) => data)
+    .catch((error) => console.log(error.response.data));
```

Comment:
Shouldn't we rethrow this error and catch it in the UI rather than logging it?

Reply:
Kind of a place holder right now until I decide how to handle it, but yes.

Reply:
Sounds good